keras_hub/src/models/segformer/segformer_image_segmenter_preprocessor.py

```diff
@@ -0,0 +1,30 @@
+import keras
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.image_segmenter_preprocessor import (
+    ImageSegmenterPreprocessor,
+)
+from keras_hub.src.models.segformer.segformer_backbone import SegFormerBackbone
+from keras_hub.src.models.segformer.segformer_image_converter import (
+    SegFormerImageConverter,
+)
+from keras_hub.src.utils.tensor_utils import preprocessing_function
+
+IMAGENET_DEFAULT_MEAN = [0.485, 0.456, 0.406]
+IMAGENET_DEFAULT_STD = [0.229, 0.224, 0.225]
+
+
+@keras_hub_export("keras_hub.models.SegFormerImageSegmenterPreprocessor")
+class SegFormerImageSegmenterPreprocessor(ImageSegmenterPreprocessor):
+    backbone_cls = SegFormerBackbone
+    image_converter_cls = SegFormerImageConverter
+
+    @preprocessing_function
+    def call(self, x, y=None, sample_weight=None):
```

Comment:
how about transformations to y - which would be masks in the training set. If the images are resized/transformed, so should the masks

Reply:
Good point. In this case, since we don't want to apply either normalization or rescaling to the masks, the image converter would just resize the images. Do we have a standard practice of defining a default converter or let the user decide this?

Reply:
for segformer, I think we should add the default resizing of mask here if we are performing resizing on images.

Reply:
Added - if a converter is present, it resizes both the images and the masks. It also rescales and normalizes the images to match the HuggingFace preprocessor

Reply:
wouldn't this also rescale the masks if rescale is part of image_converter?